api/pkg/apis/v1alpha1/vendors/stage-vendor.go

```diff
@@ -183,10 +185,22 @@ func (s *StageVendor) Init(config vendors.VendorConfig, factories []managers.IMa
 		jData, _ := json.Marshal(event.Body)
 		var status model.ActivationStatus
 		json.Unmarshal(jData, &status)
+		campaign, ok := status.Outputs["__campaign"].(string)
+		if !ok {
```

Comment:
Is this expected we just print error instead of returning error?

Reply:
Good catch. I have changed to return error.